### PR TITLE
fix: export decorators for backward compatibility

### PR DESCRIPTION
**Pull request by jannyHou**
- [x] related issue: https://github.com/strongloop/loopback/issues/3753

- [x] test cases

### CODE REVIEW

File: packages/rest/test/unit/backward-compatibility.test.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import {get} from '../..';
+import {expect} from '@loopback/testlab';
+
+describe('backward-compatibility', () => {
+  it('exports functions from @loopback/openapi-v2', async () => {
+    try {
```
Comment (raymondfeng) [inline packages/rest/test/unit/backward-compatibility.test.ts]: No `try-catch` is needed. The test passes if it complies.

File: packages/rest/test/unit/backward-compatibility.test.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      class Test {
```
Comment (raymondfeng) [inline packages/rest/test/unit/backward-compatibility.test.ts]: You need to add `/* tslint:disable-next-line:no-unused-variable */` to fix the style issue:

**Comment by jannyHou** [comment]
@slnode test please

**Comment by jannyHou** [comment]
@raymondfeng thank you for reviewing, feedbacks applied but after rebasing, module `duplexer3` fails due to a weird reason...weird prints...https://cis-jenkins.swg-devops.com/job/nb/job/loopback-next~master/label=x64%20&&%20linux%20&&%20nvm,nodeVersion=6/2317/console

Triggered a new build.

**Comment by jannyHou** [comment]
@slnode test please

**Pull request merged**